.storybook/DocsDecorator/HealthFactors/Icons/index.styles.ts

```diff
@@ -0,0 +1,12 @@
+import { css, Theme } from '@emotion/react';
+import pixelsToRem from '../../../../src/app/utilities/pixelsToRem';
```

Comment:
Think `Theme` and `pixelsToRem` can be removed here as they aren't used.

Reply:
Damn, that approval was fast 😂. Thank you for noticing this btw 👍